local/gugcat/lib.php

```diff
@@ -31,7 +31,6 @@ function local_gugcat_extend_navigation_course($parentnode, $course, $context) {
     $url = new moodle_url('/local/gugcat/index.php', array('id' => $course->id));
     $gugcat = get_string('navname', 'local_gugcat');
     $icon = new pix_icon('t/grades', '');
-    $parentnode->add($gugcat, $url, navigation_node::NODETYPE_LEAF, $gugcat, 'gugcat', $icon);
```

Comment:
If you remove this, gcat will not show here: 
![image](https://user-images.githubusercontent.com/74167293/105143455-d97ad700-5b36-11eb-86fa-8f8e935458bc.png)